backend/src/main/java/gov/cdc/usds/simplereport/service/model/CurrentOrganizationRoles.java

```diff
@@ -0,0 +1,27 @@
+package gov.cdc.usds.simplereport.service.model;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+import gov.cdc.usds.simplereport.config.authorization.OrganizationRole;
+import gov.cdc.usds.simplereport.db.model.Organization;
+
+public class CurrentOrganizationRoles {
```

Comment:
this strikes me as nearly identical to OrganizationRoles. what's the point of this class?